flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -331,6 +342,7 @@ public Builder setSnapshotProperty(String property, String value) {
      * @return {@link DataStreamSink} for sink.
      */
     public DataStreamSink<Void> append() {
+      buildProperties();
```

Comment:
This needs to build the table first, then pass the table in to build the properties map. Then it should pass the table and the properties into chainIcebergOperators.

Reply:
I've done the first two steps here, building the table first and merging the configuration. In the third step, instead of passing table and props into the chainIcebergOperators, I directly use member field in chainIcebergOperators, is this feasible? thanks.